SWLOR.Game.Server/Service/Skill.Mapping.cs

```diff
@@ -59,6 +58,7 @@ private static void LoadItemToSkillMapping()
             _itemToSkillMapping[BaseItem.ShortSpear] = SkillType.TwoHanded;
             _itemToSkillMapping[BaseItem.TwoBladedSword] = SkillType.TwoHanded;
             _itemToSkillMapping[BaseItem.DoubleAxe] = SkillType.TwoHanded;
+            _itemToSkillMapping[BaseItem.Saberstaff] = SkillType.TwoHanded;
```

Comment:
This is a good fix based on the current plan but I wanted to pick your brain about saberstaffs and twin blades. I've been considering moving both of those weapon types to the One-Handed skill given the way NWN mechanics work. I worry it gives one-handed too much weight though, so maybe there are other things we can do to improve it. It also may be a moot point with the attack roll changes you made.

Reply:
Honestly I would rather have a double weapon skill, since double weapons are interesting to balance for in regular NWN (they can't normally be finessed, but don't get the damage bonus of two handers, so they are usually worse than going dex and using two finessable weapons).  Arelith fixed this by giving an AB bonus to two handed weapons (including double weapons).  On SWLOR, saberstaffs are finessable in Legacy, but double weapons are still in an awkward spot.  (Especially since in Legacy, an offhand weapon stacked with BAB is really strong).

Making double weapons their own skill means we can adjust balance for them uniquely as needed.